src/app/pages/instances/stores/virtualization-devices.store.ts

```diff
@@ -74,6 +74,10 @@ export class VirtualizationDevicesStore extends ComponentStore<VirtualizationIns
     this.loadDevices();
   }
 
+  setSelectedInstance(instance: VirtualizationInstance): void {
```

Comment:
If there is already exists functionality to select rows, why add this method and call it in `InstanceRow`?

Reply:
If the logic was already working and rows were getting selected, why add this?

Reply:
@undsoft 
We had an issue with updating rows once state is changed (stopped, running)

Reply:
Then it would be good to fix the underlying issue with updates.

Reply:
@undsoft - that's why I added new method to update selected instance with latest updates. We need to have it updated in the store.